website/docs/r/sql_database_instance.html.markdown

```diff
@@ -240,7 +240,7 @@ The required `settings` block supports:
     for information on how to upgrade to Second Generation instances.
     A list of Google App Engine (GAE) project names that are allowed to access this instance.
 
-* `availability_type` - (Optional) This specifies whether a PostgreSQL instance
+* `availability_type` - (Optional) This specifies whether a Cloud SQL instance
```

Comment:
Seems reasonable to me! While you're here, what do you think of changing it to the following:

```
The availability type of the Cloud SQL instance, high availability (`REGIONAL`) or single zone (`ZONAL`).'
```

That displays the actual reference docs' message ("The availability type of the given Cloud SQL instance.") while still listing both options concisely.

Reply:
Definitely a concise improvement - changed per your suggestion!